pkg/configuration/configuration.go

```diff
@@ -0,0 +1,60 @@
+package configuration
+
+import (
+	"github.com/go-playground/validator/v10"
+	"github.com/spf13/viper"
+	"os"
+)
+
+const (
+	configName = "config"
+	configType = "env"
+	pathConfig = "/cmd"
+)
+
+type Config struct {
+	ConfigLogger `mapstructure:",squash"`
+	ConfigEcho   `mapstructure:",squash"`
+}
+
+type ConfigLogger struct {
+	LogLevel    string `mapstructure:"LOG_LEVEL" validate:"required"`
+	LogSaveFile bool   `mapstructure:"LOG_SAVE_FILE"`
+}
+
+type ConfigEcho struct {
+	AllowedOrigins   string `mapstructure:"ALLOWED_ORIGINS" validate:"required"`
+	AllowCredentials bool   `mapstructure:"ALLOW_CREDENTIALS"`
+	AddressEcho      string `mapstructure:"ADDRESS_ECHO" validate:"required"`
+}
+
+func LoadConfiguration() (configuration Config, err error) {
```

Comment:
unit tests for this?

Reply:
Example of tabular unit tests 
https://github.com/golang/go/blob/master/src/fmt/errors_test.go

Reply:
I added this very basic unit test for the function, as all the parameters are embedded in the file, and it's challenging to test with different parameters.

Reply:
Added test unit